### PR TITLE
WINDOW_NOT_FOUND catch

### PR DESCRIPTION
**Pull request by savkaoleg**
Sample script:

web.init();
web.setTimeout(6000);
web.open('https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_link_target');
const title = web.getTitle();
web.selectFrame('id=iframeResult');
const el = web.findElement('/html/body/a');
web.click(el);
web.pause(10000);
const hs = web.getWindowHandles();
log.info(hs);

web.selectWindow(hs[0]);
web.closeWindow();

web.click(el);

### CODE REVIEW

File: src/errors/helper.js [hunk 1]
```diff
@@ -178,6 +178,10 @@ module.exports = {
         else if (err.message === 'java.net.ConnectException: Connection refused: connect') {
             return new OxError(ERROR_CODES.WEBDRIVER_ERROR, 'The underlying WebDriver seems to have crashed: ' + err.message);
         }
+        //  no such window, target window already closed, web view not found
```
Comment (romovs) [inline src/errors/helper.js:181]: For what browsers or versions could this happen and in what situations? This should be commented.

**Review by romovs — CHANGES_REQUESTED** [review]

File: src/errors/helper.js (hunk 1 above)
Comment (savkaoleg) [reply src/errors/helper.js:181]: chrome 86

Comment (romovs) [reply src/errors/helper.js:181]: "when doing an operation on an element (e.g. click) but the window was already closed"

Comment (savkaoleg) [reply src/errors/helper.js]: Fixed

**Pull request merged**